src/pages/edit-profile/EditProfile.constants.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactElement } from 'react'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import NotificationsIcon from '@mui/icons-material/Notifications'
+import GppGoodIcon from '@mui/icons-material/GppGood'
+import { Box } from '@mui/system'
+
+import { TutorProfileTabsEnum } from '~/types'
+
+export type TutorProfileProps = {
+  [key in TutorProfileTabsEnum]: {
+    title: string
+    content?: ReactElement
+    icon?: ReactElement
+  }
+}
+
+export const tabsData: TutorProfileProps = {
+  [TutorProfileTabsEnum.Profile]: {
+    icon: <AccountCircleIcon />,
+    title: 'editTutor.main.profile',
+    content: (
+      <Box>
+        <h4>Profile</h4>
+      </Box>
+    )
+  },
+  [TutorProfileTabsEnum.Notifications]: {
+    icon: <NotificationsIcon />,
+    title: 'editTutor.main.notifications',
+    content: (
+      <Box>
+        <h4>Notifications</h4>
```

Comment:
also use `Typography` for all `Box` with `h4` `enum` variant

Reply:
No need it's just a dummy text. There will be some content for that tab

Reply:
> No need it's just a dummy text. There will be some content for that tab

ok. it's like mocked text there for future tabs content